src/Compilers/Core/Portable/Syntax/GreenNode.cs

```diff
@@ -932,16 +932,15 @@ private static bool EquivalentToInternal(GreenNode node1, GreenNode node2)
                     return SyntaxList.List(select(list[0]), select(list[1]));
                 case 3:
                     return SyntaxList.List(select(list[0]), select(list[1]), select(list[2]));
-                default:
-                {
+                default: {
```

Comment:
Please put these on the next line and indent appropriately.

Reply:
📝 The Correctness_Build failure is actually caused by this.

@HurricanKai if you ever want to run with full analyzers to see if there are warnings that could pop up during CI, you can use this command locally:

```
.\Build.cmd -a -rebuild
```

Reply:
Thanks, `-rebuild` somehow tripped it up from time to time (sometimes the xlfUpdate task didn't really work, sometimes it complained about duplicate `-build`) but `-a -warnAsError` worked perfectly for me 😄